lib/PageInteractions.pm

```diff
@@ -0,0 +1,55 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: TBD
+# Maintainer: Oleksandr Orlov <oorlov@suse.de>
+
+package PageInteractions;
+use strict;
+use warnings FATAL => 'all';
+
+use testapi;
+use parent 'Exporter';
+use Exporter;
+
+our @EXPORT = qw(
+    find
+    send_keys
+    fill_in_password
+    is_displayed
+);
+
+sub find {
+    my $self = @_;
+    assert_screen(@_);
+}
+
+sub send_keys {
+    my $key = shift;
+    my $message = shift || "Send $key key";
+    record_info('send_key', $message);
+    send_key($key, @_);
+}
+
+sub fill_in_password {
+    my $password = shift;
+    my $message = shift || "Type password";
+    my $self = @_;
+    record_info('type_password', $message);
+    type_password($password, @_);
+}
+
+sub is_displayed {
+    my $needle = shift;
+    my $timeout = shift || 10;
+    record_info('check_screen', "Check whether $needle area exists on visible screen");
```

Comment:
I guess this particular wrapper doesn't help us, as usual `check_screen` provides same level of the information.

Reply:
Again, as with other subs in this package, my poor Perl skills. Wanted to pass same arguments to record_info, but also add $message to 'is_displayed'. Did not achieve that, so left some default message, but I would like to improve that in case we'll proceed with the solution.